packages/vertica-nodejs/Makefile

```diff
@@ -4,20 +4,18 @@ connectionString=vertica://
 
 params := $(connectionString)
 
-node-command := xargs -n 1 -I file node file $(params)
+node-command := xargs -I file node file $(params)
+mocha-command := xargs -I file mocha file 
 
-.PHONY : test test-connection test-integration bench publish update-npm
+.PHONY : test test-connection test-integration test-mocha-integration test-mocha-unit bench publish update-npm
```

Comment:
nit: test-connection and publish are no longer targets in this file.

Reply:
ah yes. 'test' too
good catch